src/Providers/FindologicServiceProvider.php

```diff
@@ -47,6 +47,20 @@ public function boot(
             return;
         }
 
+        $logger = $this->getLoggerObject();
+
+        $eventDispatcher->listen(
+            'IO.Resources.Import',
+            function (ResourceContainer $container) use ($configRepository) {
+                $container->addScriptTemplate(
```

Comment:
Make sure that our snippet is only added if:

- [ ] Our search is active

- [ ] The shopkey is not empty